agents/lakeshore240/LS240_agent.py

```diff
@@ -39,8 +39,17 @@ def __init__(self, agent,
 
     # Task functions.
     def init_lakeshore_task(self, session, params=None):
-        """
-        Task to initialize Lakeshore 240 Module.
+        """Perform first time setup of the Lakeshore 240 Module.
+
+        Parameters
```

Comment:
Why the switch to numpy docstrings?

Reply:
It's the recommendation in the SO Dev Guide, I've been trying to conform to that with new documentation I write. Consistency is probably more important, but with having such little documentation, I felt like writing anything new should conform to the chosen standard, rather than perpetuating the old one, else we'd never change later.

Reply:
For consistency I switched the couple of docstrings I wrote to Google's style like most of the rest of the docstrings. Some still remain in the Sphinx style.